tests/issues/issue702.cry

```diff
@@ -0,0 +1,2 @@
+type T = { b : [32], a : [8] }
+property bug (t : T) = False
```

Comment:
Like the other test, it would probably be best if we used an example with a unique counterexample. I found that I would get the `empty FTBit` error whenever I used a non-alphabetical record type containing at least one bitvector field. So we could use, for example:
```
:sat \(x : {b:[8], a:Bit}) -> x.b == 0 /\ x.a
```